thorlcr/graph/thgraph.cpp

```diff
@@ -2516,17 +2516,13 @@ void CJobBase::init()
     thorAllocator.setown(createThorAllocator(((memsize_t)globalMemorySize)*0x100000, memorySpillAt, *logctx, crcChecking, usePackedAllocator));
 
     unsigned defaultMemMB = globalMemorySize*3/4;
-    unsigned largeMemSize = getOptUInt("largeMemSize", defaultMemMB);
```

Comment:
Must have previously been removed. Not in xsd/xslt's , environment(s) or docs.
@richardkchapman